pkg/render/apiserver.go

```diff
@@ -1686,3 +1725,103 @@ rules:
 		},
 	}
 }
+
+// clusterWideSettingsGroup returns a UISettingsGroup with the description "cluster-wide settings"
+//
+// Calico Enterprise only
+func (c *apiServerComponent) clusterWideSettingsGroup() *v3.UISettingsGroup {
+	return &v3.UISettingsGroup{
+		TypeMeta: metav1.TypeMeta{Kind: "UISettingsGroup", APIVersion: "projectcalico.org/v3"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster-settings",
+		},
+		Spec: v3.UISettingsGroupSpec{
+			Description: "cluster-wide settings",
+		},
+	}
+}
+
+// userSpecificSettingsGroup returns a UISettingsGroup with the description "user settings"
+//
+// Calico Enterprise only
+func (c *apiServerComponent) userSpecificSettingsGroup() *v3.UISettingsGroup {
+	return &v3.UISettingsGroup{
+		TypeMeta: metav1.TypeMeta{Kind: "UISettingsGroup", APIVersion: "projectcalico.org/v3"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "user-settings",
+		},
+		Spec: v3.UISettingsGroupSpec{
+			Description: "user settings",
+			FilterType:  v3.FilterTypeUser,
+		},
+	}
+}
+
+// clusterWideTigeraLayer returns a UISettings layer belonging to the cluster-wide settings group that contains all of
+// the tigera namespaces.
+//
+// Calico Enterprise only
+func (c *apiServerComponent) clusterWideTigeraLayer() *v3.UISettings {
+	namespaces := []string{
+		"tigera-compliance",
+		"tigera-dex",
+		"tigera-dpi",
+		"tigera-eck-operator",
+		"tigera-elasticsearch",
+		"tigera-fluentd",
+		"tigera-guardian",
+		"tigera-intrusion-detection",
+		"tigera-kibana",
+		"tigera-manager",
+		"tigera-operator",
+		"tigera-packetcapture",
+		"tigera-prometheus",
+		"tigera-system",
+		"calico-system",
+	}
+	nodes := make([]v3.UIGraphNode, len(namespaces))
+	for i := range namespaces {
+		ns := namespaces[i]
+		nodes[i] = v3.UIGraphNode{
+			ID:   "namespace/" + ns,
+			Type: "namespace",
+			Name: ns,
+		}
+	}
+
+	return &v3.UISettings{
+		TypeMeta: metav1.TypeMeta{Kind: "UISettings", APIVersion: "projectcalico.org/v3"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "cluster-settings.layer.tigera-infrastructure",
+		},
+		Spec: v3.UISettingsSpec{
+			Group:       "cluster-settings",
+			Description: "tigera infrastructure",
```

Comment:
nit: "Tigera Infrastructure" or "Tigera infrastructure"?

Reply:
Ah yes - to be in line with the UI:  Tigera Infrastructure, Default, User Settings, Cluster Settings